client/gutenberg/extensions/presets/jetpack/editor-shared/test/get-site-fragment.js

```diff
@@ -47,4 +47,12 @@ describe( 'getSiteFragment()', () => {
 		window._currentSiteId = 12345678;
 		expect( getSiteFragment() ).toBe( 12345678 );
 	} );
+
+	test( 'should return site slug when editing a post in Gutenberg in WP-Admin', () => {
+		delete window._currentSiteId;
```

Comment:
Should this cleanup be part of the previous test?

Reply:
There's a `beforeAll` that we should switch to a `beforeEach` so that these tests don't pollute:

https://github.com/Automattic/wp-calypso/blob/c65c99ce0fce0926855a8ec5d087db45de6e7194/client/gutenberg/extensions/presets/jetpack/editor-shared/test/get-site-fragment.js#L9-L14

Reply:
Good point, will do the change.